### DISCUSSION TITLE
Real instances problem

### DISCUSSION THREAD

**Discussion by Whhhatttt** [JacquesLucke/animation_nodes#1954] [Q&A]
I'm trying to make an animation. I'm using Object Instancer. The problem is that with a large number of instances, the computer and blender in general begins to lag. Is there any way to bypass the creation of real objects? For example, the same system as in geometry nodes... Instance on points.. In this case, real objects are not created. What should I do?

**Comment by OmarEmaraDev** [comment]
See the use of the Unity Triangle node: https://docs.animation-nodes.com/release_notes/v2_1/mesh/#unity-triangle
